//----------------------------------------------------------------------------
// Copyright 2023, Ed Keenan, all rights reserved.
//----------------------------------------------------------------------------

syntax = "proto3";

option optimize_for = LITE_RUNTIME;

message Address_proto
{
	// Define Message here:  
	optional bytes poName = 1;
	optional bytes poAddr = 2;
	optional bytes poPhone = 3;
}

// --- End of File ---
